locales/ru.json

```diff
@@ -0,0 +1,8 @@
+{
+	"commands": {
+    "notFound": "Команда {name} не найдена.",
+    "ui": {
+      "create": "Создать команду"
+    }
+  }
+}
```

Comment:
The same with newlines.

Reply:
I think json shouldn't contain newlines.

Reply:
It's the rule for any text file from POSIX. You can read:

1. https://thoughtbot.com/blog/no-newline-at-end-of-file
2. https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
3. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Sure, i'll create new PR with eslint and EditorConfig after merge that PR.